tools/code_review_tools/taskcluster.py

```diff
@@ -86,7 +86,7 @@ def auth(self, client_id=None, access_token=None):
                 "clientId": client_id,
                 "accessToken": access_token,
             }
-            self.options["rootUrl"] = "https://taskcluster.net"
+            self.options["rootUrl"] = "https://community-tc.services.mozilla.com"
```

Comment:
This should be firefox-ci, right?

Reply:
I would agree to firefox-ci here, this should be used only when providing directly TC credentials, meaning a code-review dev trying to debug a task.